### DISCUSSION TITLE
Docker compose installation tutorial for Synology NAS (and beginner)

### DISCUSSION THREAD

**Discussion by EricGuic** [mediacms-io/mediacms#282] [General]
Hi (from a french follower),
I'm not a developper, and as a growing number of people, I start to take back control over my data using simple services that I host on my Synology NAS using Docker.
Folowing tutorials (likes the one provided by [Marius Hosting](https://mariushosting.com/), or [Rusty](https://www.blackvoid.club/) or many other...) I've successfully installed Portainer to manage simple services like Snapdrop, Tiny RSS, AdGuardHome...

So now, I want to host my own video platform (I'm a director, and don't want to use Youtube or Vimeo, and want to try MediaCMS to host my files, and generate multiples resolution and the embed code needed to add those videos to my blogs under Ghost and Wordpress).

But in your documentation or in the medium post you write, I see installation process that are too complicated for the kind of user I represent.

Is there a way to write a guide, that will describe : 

- how to fully install MediaCMS using the stack feature inside Portainer (the compose file is written directly inside the Portainer web UI, and is relatively simple to update the image later or to edit the configuration variable inside this compose syntax without the need to type ssh command)
- how to map the directory so the media files can be store outside the container, on a volume on the NAS (this is for example what I do for my Plex Media Server : the app is inside a container in /docker/pms/ and the media are inside multiple folders inside media/movie or media/music and so on.

Thanks per advance if you can provide this sort of help.

Best regards from France. Eric

**Comment by siousu** [comment]
Dear All, I'm trying to achieve the same as Eric! :) I've a Synology NAS and would like to use the docker from here: https://registry.hub.docker.com/r/mediacms/mediacms/ 

But so far I couldn't get this docker image running. Does anyone have some tips?

Thanks a lot and BR,
Simon
